osmdata.cpp

```diff
@@ -1,9 +1,10 @@
+#include <cstdio>
```

Comment:
I would consider <cstdio> a C++ header. The C header is called <stdio.h>. There is a subtle difference between two, for example, the former exports namespaces.